src/test/kotlin/no/nav/syfo/dialogmote/GetDialogmoteEnhetApiSpek.kt

```diff
@@ -0,0 +1,165 @@
+                val urlEnhetAccess = "$dialogmoteApiBasepath$dialogmoteApiEnhetUrlPath/${ENHET_NR.value}"
+                val urlEnhetNoAccess = "$dialogmoteApiBasepath$dialogmoteApiEnhetUrlPath/${ENHET_NR_NO_ACCESS.value}"
+                val validToken = generateJWT(
+                    environment.loginserviceClientId,
+                    wellKnownVeileder.issuer,
+                    VEILEDER_IDENT,
+                )
+                describe("Happy path") {
+
+                    val newDialogmote = generateNewDialogmote(ARBEIDSTAKER_FNR)
+                    val newDialogmoteAdressebeskyttet = generateNewDialogmote(ARBEIDSTAKER_ADRESSEBESKYTTET)
+                    database.connection.use { connection ->
+                        connection.createDialogmoteWithReferences(
+                            newDialogmote = newDialogmote
+                        )
+                        connection.createDialogmoteWithReferences(
+                            newDialogmote = newDialogmoteAdressebeskyttet
+                        )
+                    }
+
+                    it("should return DialogmoteList if request is successful") {
+                        with(
+                            handleRequest(HttpMethod.Get, urlEnhetAccess) {
+                                addHeader(Authorization, bearerHeader(validToken))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+
+                            val dialogmoteList = objectMapper.readValue<List<DialogmoteDTO>>(response.content!!)
+
+                            dialogmoteList.size shouldBeEqualTo 1
+
+                            val dialogmoteDTO = dialogmoteList.first()
+                            dialogmoteDTO.tildeltEnhet shouldBeEqualTo newDialogmote.tildeltEnhet
+                            dialogmoteDTO.arbeidstaker.personIdent shouldBeEqualTo newDialogmote.arbeidstaker.personIdent.value
+                            dialogmoteDTO.arbeidsgiver.virksomhetsnummer shouldBeEqualTo newDialogmote.arbeidsgiver.virksomhetsnummer.value
+
+                            dialogmoteDTO.tidStedList.size shouldBeEqualTo 1
+                            val dialogmoteTidStedDTO = dialogmoteDTO.tidStedList.first()
+                            dialogmoteTidStedDTO.sted shouldBeEqualTo newDialogmote.tidSted.sted
+                        }
+                    }
+                }
+
+                describe("Unhappy paths") {
+                    it("should return status Unauthorized if no token is supplied") {
+                        with(
+                            handleRequest(HttpMethod.Get, urlEnhetAccess) {}
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.Unauthorized
+                        }
+                    }
+
+                    it("should return status Forbidden if denied access to person") {
```

Comment:
´access to person´ -> ´access to Enhet´?

Reply:
Fikset.